alexandria/core/serializers.py

```diff
@@ -37,16 +39,39 @@ class Meta:
         )
 
 
-class CategorySerializer(BaseSerializer):
+class SlugModelSerializer(BaseSerializer):
+    """
+    Ensure on creation that the model will recevie a slug.
```

Comment:
```suggestion
    Ensure on creation that the model will receive a slug.
```

Reply:
Funny that gh thinks "Ensure" is a class...

Reply:
Guess only in the suggested-change view :)